android/sdk/src/main/java/org/jitsi/meet/sdk/VideoCodecMimeType.java

```diff
@@ -0,0 +1,19 @@
+package org.jitsi.meet.sdk;
+
+/** Enumeration of supported video codec types. */
+public enum VideoCodecMimeType {
+    VP8("video/x-vnd.on2.vp8"),
+    VP9("video/x-vnd.on2.vp9"),
+    H264("video/avc"),
+    AV1("video/av01");
+
+    private final String mimeType;
+
+    private VideoCodecMimeType(String mimeType) {
+        this.mimeType = mimeType;
+    }
+
+    String mimeType() {
```

Comment:
getMimeType, if we want to be very rigorous :)

Reply:
I copied this over from WebRTC ;-)